client/components/application-summary.js

```diff
@@ -73,6 +99,29 @@ class ApplicationSummary extends React.Component {
     return !section.show || section.show(this.props.values);
   }
 
+  getComments = subsection => {
+    let newComments = 0;
+
+    const repeaters = {
+      protocols: /^protocol\./,
+      establishments: /^establishments\./,
+      poles: /^pole\./,
+      'action-plan': /objectives\./
+    }
```

Comment:
Is it possible to extract this from the schema? I'm mildly concerned it might differ across schema versions.